kernel/kernel-benchmarks/src/test/java/io/delta/kernel/benchmarks/WorkloadBenchmark.java

```diff
@@ -98,6 +98,7 @@ public static void main(String[] args) throws RunnerException, IOException {
     Options opt =
         new OptionsBuilder()
             .include(WorkloadBenchmark.class.getSimpleName())
+            .shouldFailOnError(true)
```

Comment:
does this mean that one spec failure will cause all others to fail?

any idea how this would impact a downstream consumer of these reports?